### ISSUE TITLE
Can it be used with CA Appdata Backup/Restore app?

### ISSUE THREAD

**Issue by paulmcclellan** [Glasti1/unraid-plex-backup-fine-tuning#4]
I just implemented this, and was very pleased with how quick the db backup was!  Question: Can I use it in conjunction with CA Appdata Backup, as it also allows for easy restores?

**Comment by Glasti1** [comment]
Yes you can use the CA appdata backup plugin if you prefer. I would setup a costum script to backup the metdata folders just incase of drive failure.

**Comment by paulmcclellan** [comment]
Thanks.  I do have a question about the container configuration, specifically the Container Path.  In your example you used '/config/app/Plex Media Server/Cache/Transcode/Sync+'  I'm having trouble figuring out what value to use here?

**Comment by Glasti1** [comment]
> Thanks. I do have a question about the container configuration, specifically the Container Path. In your example you used '/config/app/Plex Media Server/Cache/Transcode/Sync+' I'm having trouble figuring out what value to use here?

it looks like this in my current config
![image](https://user-images.githubusercontent.com/57273176/113485578-8fa35b00-94ae-11eb-80a8-1baee541c776.png)
the path can be different depending which container you use. i use hotio/plex

**Comment by paulmcclellan** [comment]
Thanks.  I appreciate your help.  Hopefully last questions: Does your container path point to a share?  Would it be something like 'mnt/appdata/Plex/config/Cache/Transcode/Sync+'?

**Comment by Glasti1** [comment]
in 6.8.3 i made an Metadata share where it goes in. `/mnt/user/metadata/`
And this share uses the appdata cache drive.

This is a little different in my 6.9 setup. I created a pool for plex with a 500GB nvme. That pool i named `appdata_plex`. in that pool i created a user share called `Plex`. In this share i have the folder called`plex_data` where all the folders are in that are in the guide. And another folder called `Plex` which is the plex appdata folder.

So, `/mnt/appdata_plex/plex/plex_data` and `/mnt/appdata_plex/plex/plex/`

**Comment by paulmcclellan** [comment]
Thanks for your help with this.  I made a tweak, where am leaving all the data in the original locations, but still backing them up separately.  To do this, I modified the 'plexdata' backup script, and added separate tars for the Cache, Media and Metadata directories.  To my tiny brain, this was an easier solution.  

I do appreciate your help with this.  It has accomplished the goal of reducing the Plex downtime for backing up appdata!
